test/translator/inquiry_test.cc

```diff
@@ -315,4 +315,27 @@ TEST(TranslateUnitSerialNumberVpd, neither) {
   ASSERT_STREQ(product_serial_number, formatted_hex_string);
 }
 
+TEST(TranslateBlockLimitsVpd, success) {
```

Comment:
Can we add tests for the different possible contents of identify_controller_data?